.travis.yml

```diff
@@ -42,5 +42,8 @@ script:
   # C++ tests.
   - build/tests/bag_tests --samples-path examples/sample-data
   # Python tests.
-  - LD_LIBRARY_PATH=/home/travis/build/jice73/BAG/build/api PYTHONPATH=/home/travis/build/jice73/BAG/build/api/swig_i python python/test_all.py
+  # HOME == /home/travis
+  # ? == build/jice73 (some kind of git/checkout/github variable?)
+  - env
+  #- LD_LIBRARY_PATH=/home/travis/build/jice73/BAG/build/api PYTHONPATH=/home/travis/build/jice73/BAG/build/api/swig_i python python/test_all.py
```

Comment:
Doesn't having 'jice73' makes it only working with your Travis account?

Reply:
It does.  There is no mention of jice73 in the latest version of the.travis.yml file.  :)

Reply:
I am confused by the commented rows! What is the point of keeping them?

Reply:
They aren't kept; that's an old version. of the file.